ext/session/session.c

```diff
@@ -1388,6 +1389,12 @@ static zend_result php_session_send_cookie(void)
 		return FAILURE;
 	}
 
+	/* Check for invalid settings combinations */
+	if (UNEXPECTED(PS(cookie_partitioned) && !PS(cookie_secure))) {
+		php_error_docref(NULL, E_WARNING, "Partitioned session cookie cannot be used without also configuring it as secure");
+		return FAILURE;
+	}
```

Comment:
I'm wondering if it makes sense to have a dedicated INI handler to check the partionned is only true when secure is true, but if the secure is modifier after there is no way to check this.... so maybe this is fine.

Reply:
Yeah, I'm ACK on ext/session :)